docs/source/changelog.rst

```diff
@@ -4,11 +4,19 @@ Changes in IPython Parallel
 ===========================
 
 
-5.3
+6.0
 ---
 
+This is a major release due to a compatibility change, but not a big one.
+The main compatibility change is that timestamps everywhere are now timezone-aware UTC timestamps.
+This means you may see comparison errors if you have code that uses datetime objects without timezone info (so-called naïve datetime objects).
+
+Other fixes:
+
 - Rename :meth:`Client.become_distributed` to :meth:`Client.become_dask`.
   :meth:`become_distributed` remains as an alias.
+- import joblib from a public API for joblib backend
```

Comment:
Is this change to take advantage of using https://pythonhosted.org/joblib/parallel.html#custom-backend-api-experimental ?